### ISSUE TITLE
ubuntu install error

### ISSUE THREAD

**Issue by Dongxueyang** [GeoscienceAustralia/anuga_core#201]
Hi guys, I got an error when I install the anuga in Ubuntu.
When I run " sudo bash tools/install_ubuntu.sh", I got this error  " Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-hzu_axsv/netCDF4/" 
Can anybody help me?
Thanks a lot
dong

**Comment by stoiver** [comment]
@Dongxueyang 

You should go through the installation script one line at a time to isolate the error. Though it looks like there is an error with setting up netCDF4. 

It would be useful to get a more detailed error message.

**Comment by Dongxueyang** [comment]
@stoiver
Thanks a lot. I have solve the problem. This error is relative to the version of pip. 
But when I installed successfully(I run "sudo sh tools/install_ubuntu.sh" without any error)
I try to test and run " python runtest.py" 
I got another error like this:

dingwentao@dingwentao:~/Software/anuga_core-master$ python runtests.py 
Building, see build.log...
Build OK
Running unit tests for anuga
NumPy version 1.16.4
NumPy relaxed strides checking option: True
NumPy is installed in /usr/local/lib/python2.7/dist-packages/numpy
Python version 2.7.6 (default, Nov 13 2018, 12:45:42) [GCC 4.8.4]
nose version 1.3.7
....../usr/local/lib/python2.7/dist-packages/netCDF4/__init__.py:3: UserWarning: 
WARNING: Backwards incompatible files will be created with HDF5 1.10.x 
and netCDF < 4.4.1. Upgrading to netCDF4 >= 4.4.1 or downgrading to 
to HDF5 version 1.8.x is highly recommended 
(see https://github.com/Unidata/netcdf-c/issues/250).
  from ._netCDF4 import *
Segmentation fault (core dumped)

Is this the problem of version of libs?
Hope for your reply

Best regards
 dong
